docs/fundamentals/jcloud/advanced.md

```diff
@@ -73,6 +73,101 @@ executors:
           type: efs
 ```
 
+### GPU support
+
+We have GPU support in JCloud. We have `2` kind of GPU usages, `shared` or `dedicated`.
+
+```{note}
+Due to enable GPU support may require to cluster to dynamically provision new GPU nodes. It may take extra 2-3 mins till all flows are running in the cluster.
+```
+
+#### Shared GPU
+
+Shared GPU is to enable time-slicing feature, which is used to allow workloads that land on oversubscribed GPUs to interleave with one another. In that case, executor may share a GPU with others(10 executor at maximum).
+
+```yaml
+jtype: Flow
+protocol: http
+executors:
+- name: sb0
+  uses: jinahub+docker://CustomExecutor
+  jcloud:
+      resources:
+        gpu: shared
+```
+
+```{note}
+Please notice, nothing special is done to isolate workloads that are granted replicas from the same underlying GPU, and each workload has access to the GPU memory and runs in the same fault-domain as of all the others (meaning if one workload crashes, they all do).  
+```
+
+#### Dedicated GPU
+
+Dedicated GPU is a normal way to provision GPU to the executor, it will automatically create nodes or assign the executor landing on GPU nodes. In this case, executor own the whole GPU, minimum GPU core is `1`, `max`maximum is `4`.
```

Comment:
It says 'GPU cores', is that correct? I imagine it is between 1 and 4 whole GPUs?
```suggestion
Using a dedicated GPU is a default way to provision GPU to the Executor. This will automatically create nodes or assign the Executor to land on a GPU node. In this case, executor owns the whole GPU. You can assign between 1 and 4 GPUs.
```

Reply:
yes, it's correct. dedicated GPU means you can have the whole GPU.